components/util/consulting/benefits.module.css

```diff
@@ -0,0 +1,13 @@
+.description {
```

Comment:
utility classes over custom css

Reply:
Was trying to change elements' styles inside `<TinaMarkdown />`
1. Add another prose-* in the tailwind.config.js file
2. Add custom CSS

I choose the second option since it is closer to editing in the future.